corehq/apps/data_dictionary/urls.py

```diff
@@ -11,7 +12,8 @@
     url(r"^generate/$", generate_data_dictionary),
     url(r"^json/$", data_dictionary_json, name='data_dictionary_json'),
     url(r"^json/?(?P<case_type_name>\w+)/?$", data_dictionary_json, name='case_type_dictionary_json'),
-    url(r"^update_case_property/$", update_case_property, name='update_case_property'),
+    url(r"^update_case_properties/$", update_case_property, name='update_case_property'),
+    url(r"^update_case_property/$", update_case_property_description, name='update_property_description'),
```

Comment:
How about making these two regexes `r"^update_case_property/$"` and `r"^update_case_property_description/$"`